src/styles/index.scss

```diff
@@ -0,0 +1,47 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600;700;800&family=Roboto+Slab:wght@300;400;500;600&display=swap');
+@import './navbar.scss';
+@import './book.scss';
+@import './input.scss';
+@import './categories.scss';
+
+:root {
+  --bubble-gum-pink: #ff5ab7;
+  --black: #1e1e1e;
+  --pinkish-grey: #c7c7c7;
+  --very-light-pink: #c4c4c4;
+  --pale-grey: #f5f6fa;
+  --black-5: rgba(0, 0, 0, 0.05);
+  --z: #fafafa;
+  --black-two: #121212;
+  --white: #e8e8e8;
+  --white-two: #fff;
+  --medium-blue: #307bbe;
+  --cool-blue: #4386bf;
+  --azure: #0290ff;
+}
+
+.Bookstore-CMS {
+  width: auto;
+  height: auto;
+  padding: 3.313rem 3.75rem 7.688rem 3.75rem;
+  background-color: var(--pale-grey);
+  justify-content: center;
+
+  .panel-bg {
+    width: 87.5rem;
+    height: auto;
+    margin: 0 2.5rem 2.063rem 0;
+    padding: 0 0 4.625rem;
+    border-radius: 4px;
+    box-shadow: 0 0 90px 0 var(--black-5);
+    background-color: var(--z);
+
+    .horizontal-divider {
+      width: 76rem;
```

Comment:
As previously requested, the horizontal scroll issue has not been resolved, kindly resolve this issue as it makes the UI quite distorted and provides an overall bad UX.

See the video below.



https://user-images.githubusercontent.com/84930355/222921587-604664da-d507-4731-9a73-987144e65269.mp4